linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitor.java

```diff
@@ -93,7 +93,12 @@ public static void main(String[] arguments)
     String bomCoordinates = arguments[0];
     List<String> coordinatesElements = Splitter.on(':').splitToList(bomCoordinates);
 
-    new LinkageMonitor().run(coordinatesElements.get(0), coordinatesElements.get(1));
+    try {
+      new LinkageMonitor().run(coordinatesElements.get(0), coordinatesElements.get(1));
+    } catch (LinkageMonitorException ex) {
```

Comment:
This fixes the immediate issue. Looking at this now though, I don't think this should be an exception at all. It is not an exceptional or unexpected condition, anymore than a compiler noticing a syntax error would be. We might want to change this to a return value as a list of linkage errors or some such thing.

Reply:
Updated `LinkageMonitor.run` to return new problems.